solr/solr-ref-guide/src/shard-management.adoc

```diff
@@ -205,7 +261,61 @@ Shards can only created with this API for collections that use the 'implicit' ro
 
 Use SPLITSHARD for collections created with the 'compositeId' router (`router.key=compositeId`).
 
-`/admin/collections?action=CREATESHARD&shard=_shardName_&collection=_name_`
+[.dynamic-tabs]
+--
+
+[example.tab-pane#v1createshard]
+====
+[.tab-label]*V1 API*
+
+*Input*
+
+[source,text]
+----
+http://localhost:8983/solr/admin/collections?action=CREATESHARD&shard=_shardName_&collection=_name_
```

Comment:
[0] It might be cool if the v1 and v2 examples here used the same param vals so they were truly equivalent.